annot.go

```diff
@@ -295,3 +308,30 @@ func write(writer io.Writer, annots []*Annot) error {
 
 	return nil
 }
+
+func writeArrowOrRange(annots []*Annot) string {
+	widthWritten := 0
+
+	b := &strings.Builder{}
+
+	for _, a := range annots {
+		if a.ColEnd == 0 {
+			b.WriteString(strings.Repeat(" ", a.pipeColIdx-widthWritten))
+			b.WriteString("↑")
+			widthWritten = a.pipeColIdx + 1
+		} else {
+			b.WriteString(strings.Repeat(" ", a.Col-widthWritten))
+			if a.Col == a.pipeColIdx {
+				b.WriteString("├")
+			} else {
```

Comment:
This multiple  if-else branching is complicated and it's uneasy to see what it does

Could you move some of them on simpler functions maybe?

Reply:
I'm also fan of early returns and less else. In this case I removed the parent if else with a `continue` statement.

For this particular statement I don't see any simplification. It would pack the logic in a box with self written note on it (function name). I try to avoid logic indirection behind custom names.

Reply:
OK, everyone has his coding preferences 😅😂🤣

Reply:
Thanks for your input. Putting something in a box and label it, does not always help readability. Indirection and an imaged name by the author can complicate things. In this case better an ugly else as a own made up name. An example would help to understand what could be done.